tests/conftest.py

```diff
@@ -45,15 +45,28 @@ def local_grand_challenge() -> Generator[str, None, None]:
                 "scripts/development_fixtures.py",
                 "scripts/algorithm_evaluation_fixtures.py",
                 "scripts/image10x10x10.mha",
+                "scripts/minio.py",
```

Comment:
I assume an unrelated fix for the problem at hand?

Reply:
This is now required for `make development_fixtures`, was changed a couple of weeks ago.